src/main/java/goral/psychotherapistoffice/web/AboutMeController.java

```diff
@@ -1,16 +1,29 @@
 package goral.psychotherapistoffice.web;
 
 
+import goral.psychotherapistoffice.domain.counter.CounterService;
+import jakarta.servlet.http.HttpServletRequest;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
 @Controller
 public class AboutMeController {
 
+    private final CounterService counterService;
+
+    public AboutMeController(CounterService counterService) {
+        this.counterService = counterService;
+    }
+
+
     @GetMapping("/omnie")
-    public String therms(Model model){
+    public String therms(Model model, HttpServletRequest request){
+        counterService.httpParameterToIncrement(request);
```

Comment:
you added `counterService.httpParameterToIncremen` to all controllers. when i had small experience i make same

you can try create one for them :)

https://www.baeldung.com/spring-mvc-handlerinterceptor#Interceptor
```java
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import org.springframework.web.servlet.HandlerInterceptor;
import org.springframework.web.servlet.ModelAndView;

public class PageVisitInterceptor implements HandlerInterceptor {

    private final CounterService counterService;

    public AboutMeController(CounterService counterService) {
        this.counterService = counterService;
    }

    @Override
    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler)
            throws Exception {
        return true;
    }

    @Override
    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler,
            ModelAndView modelAndView) throws Exception {
        counterService.httpParameterToIncrement(request);
    }

    @Override
    public void afterCompletion(HttpServletRequest request, HttpServletResponse response, Object handler,
            Exception ex) throws Exception {
       //empty function
    }
}
```